src/components/SingleReview.js

```diff
@@ -10,12 +11,13 @@ const SingleCategory = (props) => {
     }, [id]);
     return (
     <ul>
-        <li key={review.title} class="review-box">
+        <li key={review.title} className="single-review">
             <img className="review-image" src={review.review_img_url} alt={review.title}/>
             <p>{review.title}</p>
-            <p class="designer">{review.designer}</p>
+            <p className="designer">{review.designer}</p>
```

Comment:
Same as above with props! You could do something like const {designer, review_body} = review, and that way you wouldn't have to keep typing "review."